ml-agents/mlagents/trainers/trainer/trainer.py

```diff
@@ -109,20 +107,12 @@ def reward_buffer(self) -> Deque[float]:
         """
         return self._reward_buffer
 
-    @timed
-    def save_model(self, name_behavior_id: str) -> None:
-        """
-        Saves the model
-        """
-        self.get_policy(name_behavior_id).save_model(self.get_step)
-
-    def export_model(self, name_behavior_id: str) -> None:
+    @abc.abstractmethod
+    def save_model(self) -> None:
         """
-        Exports the model
+        Saves model file(s) for the policy or policies associated with this trainer.
```

Comment:
This docstring is different from the behavior of save_model in rl_trainer as the rl_trainer will only "save the first" policy but this comment indicates that it will save the policie**s**

Reply:
We don't have any implementations at this time where we actually need to save more than one policy.  In the future I could anticipate that case (thus this docstring).  I guess the change here should depend on our decision about how to handle `Trainer.policy` vs. `Trainer.policies`